py/obj.h

```diff
@@ -426,6 +426,9 @@ extern const mp_obj_type_t mp_type_MemoryError;
 extern const mp_obj_type_t mp_type_NameError;
 extern const mp_obj_type_t mp_type_NotImplementedError;
 extern const mp_obj_type_t mp_type_OSError;
+#if MICROPY_PY_BUILTINS_TIMEOUTERROR
```

Comment:
No need for this #if guard (it's not generally used for optional header declarations to keep things cleaner).